biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -19,16 +19,26 @@ import { SimulationDispatchSpec } from '@biosimulations/dispatch/datamodel';
 import { v4 as uuid } from 'uuid';
 import path from 'path';
 import * as csv2Json from 'csv2json';
+import {
+  Cron,
+  CronExpression,
+  Interval,
+  SchedulerRegistry,
+  Timeout,
+} from '@nestjs/schedule';
+import { CronJob } from 'cron';
 
 @Controller()
 export class AppController {
   constructor(
     private readonly configService: ConfigService,
     private hpcService: HpcService,
     private sbatchService: SbatchService,
-    @Inject('DISPATCH_MQ') private messageClient: ClientProxy
+    @Inject('DISPATCH_MQ') private messageClient: ClientProxy,
+    private schedulerRegistry: SchedulerRegistry
   ) {}
   private logger = new Logger(AppController.name);
```

Comment:
Both of these values will be the same instance if the logger. Is that the intention?